### PR TITLE
New and modified templates

### PR DESCRIPTION
**Pull request by VitalRus95**
**Modified**: ifthen, ifelse, for, while, repeat (indentation set to 1 tab and added inside the templates, e.g. after `then` or `else`).
**Added**: wtrue, wgosub, rfalse, rgosub, bthold, kbhold.

Overview of new templates:
- wtrue = while true
```
while true
    wait 0

    |
end // while true
```
- rfalse = repeat until false
```
repeat
    wait 0

    |
until false // repeat until false
```
- wgosub = while gosub
```
while gosub |
    wait 0

    
end // while gosub
```
- rgosub = repeat until gosub
```
repeat
    wait 0

    
until gosub | // repeat until gosub
```
- bthold = Hold ingame button
```
if // Insert the number of the ingame button to hold below
    00E1:   player 0 pressed_key |
then
    if // Insert the intended milliseconds threshold below
        timera > 
    then
        
    end
else
    timera = 0
end // if
```
- kbhold = Hold keyboard button
```
if // Insert the number of the keyboard button to hold below
    0AB0:   is_key_pressed |
then
    if // Insert the intended milliseconds threshold below
        timera > 
    then
        
    end
else
    timera = 0
end // if
```

### CODE REVIEW

File: templates.txt [hunk 1]
```diff
@@ -44,3 +73,28 @@ hex=HEX..END construct
 = |
 =end // hex
 
+bthold=Hold ingame button template
+=if // Insert the number of the ingame button to hold below
+=	00E1:   player 0 pressed_key |
```
Comment (x87) [inline templates.txt:78]: this is a game-specific opcode. In LCS/VCS 00E1 has different meaning. 

Can you move this template and `kbhold` to gta3/vc/sa templates?

Comment (VitalRus95) [reply templates.txt:78]: Thanks for this detail! Sure, I'll change it as soon as possible!

Comment (x87) [reply templates.txt]: You can also implement those templates for LCS/VCS using appropriate opcodes

Comment (VitalRus95) [reply templates.txt]: I've never worked with LCS and VCS, but I hope I chose the right opcodes for `bthold` template. :)

**Comment by x87** [comment]
@VitalRus95 Thank you!

**Pull request merged**